package.json

```diff
@@ -4,50 +4,70 @@
   "description": "Fiat and non-fiat tokens part of the CENTRE network.",
   "main": "truffle-config.js",
   "directories": {
-    "lib": "lib",
     "test": "test"
   },
   "scripts": {
-    "dev": "lite-server",
-    "truffle-test": "scripts/start-ganache.sh && truffle test",
-    "test": "scripts/start-ganache.sh && ./node_modules/.bin/solidity-coverage"
+    "setup": "git config core.hooksPath .githooks",
+    "compile": "rm -rf './build' && truffle compile",
+    "typechain": "yarn compile && rm -rf './@types/generated' && typechain --target=truffle-v5 --outDir './@types/generated' 'build/contracts/**/*.json'",
+    "typecheck": "tsc -p . --noEmit",
+    "lint": "eslint --ext '.js,.ts' './**/*.{j,t}s'",
+    "fmt": "prettier --write './**/*.sol' './**/*.js' './**/*.ts' './**/*.json' './**/*.md'",
+    "ganache": "ganache-cli --accounts=15 --deterministic --defaultBalanceEther=1000000 --quiet",
+    "test": "truffle test",
+    "coverage": "truffle run coverage",
+    "solhint": "solhint 'contracts/**/*.sol'",
+    "precommit": "yarn typechain && yarn fmt && yarn typecheck && yarn lint && yarn solhint"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/centrehq/centre-tokens.git"
   },
   "keywords": [],
   "author": "",
-  "license": "ISC",
+  "license": "MIT",
```

Comment:
Why this? They're equivalent, except that ISC removes a couple of words.

Reply:
Because the license text in the source file and the LICENSE file is MIT not ISC